test/release/examples/primers/modules.chpl

```diff
@@ -94,81 +99,83 @@ module MainModule {
   proc main() {
     writeln("Access from outside a module");
 
-    /* If a module is not the main module for a program, it is desirable for its
+    /* Access From Outside a Module
+       ----------------------------
+       If a module is not the main module for a program, it is desirable for its
        contents to be accessible to external modules.  There are several
        strategies for accomplishing this:
 
        First, a symbol can be referenced explicitly - this is done using the
-       module name and a separating '.' as a prefix to the name of the symbol
+       module name and a separating ``.`` as a prefix to the name of the symbol
        desired.
      */
-    var thriceFoo = 3 * modToUse.foo; // should be 36
+    var thriceFoo = 3 * modToUse.foo; // should be ``36``
     writeln(thriceFoo);
 
-
+    /* If several of the module's symbols are desired, or the same symbol is
+       desired multiple times, then it can be convenient to utilize what is
+       known as a "use statement".
+    */
     {
```

Comment:
Yes.  I've scoped every section so that we don't run into naming conflicts, inappropriate shadowing, and the unnecessary bleeding of symbols.  I might be tempted to make more of the inner comments "code comments" instead of paragraphs so that it doesn't get as broken up, though